python/pyspark/sql/readwriter.py

```diff
@@ -545,6 +545,57 @@ def partitionBy(self, *cols):
         self._jwrite = self._jwrite.partitionBy(_to_seq(self._spark._sc, cols))
         return self
 
+    @since(2.2)
+    def bucketBy(self, numBuckets, *cols):
+        """Buckets the output by the given columns on the file system.
```

Comment:
So the `bucketBy` description in the scaladoc is a bit more in depth, you might just want to copy that.

Also generally our style for multi-line doc string is to have the open `"""` on its own line.